plugins-root/check_icmp.c

```diff
@@ -50,19 +50,24 @@ const char *email = "devel@monitoring-plugins.org";
 #if HAVE_SYS_SOCKIO_H
 #include <sys/sockio.h>
 #endif
+
+/*
```

Comment:
That's bad practice. Either remove it or keep it, but commenting out code should be avoided. That's what we have git for.